src/auxinfo/participant.rs

```diff
@@ -75,6 +74,7 @@ pub(crate) struct AuxInfoParticipant {
 }
 
 impl ProtocolParticipant for AuxInfoParticipant {
+    type Input = ();
```

Comment:
thought: I think we want to enforce that `AuxInfo` is tied to a specific key; eventually this input should reflect that. For example, maybe it'll be public key shares so we can tie them into the proof transcripts.

Not something to address now, but maybe I'll add an issue to #149.

Reply:
Hmm, my draft PR #231 doesn't seem to need `Input` at all, so maybe I'm missing something?

Where in `auxinfo` are things tied to a specific key? I see `new_auxinfo` but that's part of the protocol execution.

Reply:
Yeah, the implementation of auxinfo is not tied to a specific key right now. But in the protocol description (Figure 3, Key-Generation step 2), auxinfo is run on input that includes the public key shares **X**.

As far as I can tell, key material in auxinfo is only used as proof context (so we're not missing anything in the auxinfo output calculations that would require **X**). This is a spot where there's some inconsistent notation across different figures in the paper:
- in Fig 3, auxinfo takes several input parameters, including `sid`, `rid`, and **X**, 
- in Figure 6 it takes only `ssid` as a parameter

I think Figure 6 is badly notated because `ssid` as it's defined everywhere else in the paper includes the outputs of auxinfo, so there's a circular dependency in there. But within auxinfo, that `ssid` input parameter is used as a unique ID for messages (which we cover with `Identifier`) and as proof context.

Reply:
Got it. We can address this in #231.